template/src/constant/index.tsx

```diff
@@ -0,0 +1,5 @@
+import { Dimensions } from 'react-native';
+
+export const WIDTH = Dimensions.get('window').width;
+export const HEIGHT = Dimensions.get('window').height;
+export const IMAGESIZE = Dimensions.get('window').width;
```

Comment:
Please remove this constants as they aren't really constant (can change with screen orientations). Also, instead of the `Dimensions`, we must use `useDimensions` hook.